src/main/java/com/alkemy/ong/model/Member.java

```diff
@@ -0,0 +1,48 @@
+package com.alkemy.ong.model;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+
+@Entity
+@Table(name = "members")
+@Setter
+@Getter
+@SQLDelete(sql = "UPDATE members SET deleted = true WHERE id = ?")
+@Where(clause = "deleted = false")
+public class Member {
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE)
+    private Long id;
+
+    @Column(nullable = false)
+    private String name;
+
+    @Column(name = "facebook_url", nullable = false)
```

Comment:
Todos los atributos que son nullable = false deberian tener un mensaje de error, esto lo podemos hacer mediante annotations tambien. Pero las redes sociales no estan como campos NOT NULL, justamente si podrian serlo. Unicamente el name y la image serian NOT NULL